lib/lightspeed_restaurant/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module LightspeedRestaurantClient
-  VERSION = '3.4.0'
+  VERSION = '3.5.0'
```

Comment:
same. keep it out for now please so we make the distinction between work & release 🙏